packages/ckeditor5-engine/src/model/nodelist.ts

```diff
@@ -247,13 +247,13 @@ export default class NodeList implements Iterable<Node> {
  * occupy multiple items if its offset size is greater than one.
  */
 function makeOffsetsArray( nodes: Array<Node> ): Array<Node> {
-	const offsets: Array<Node> = [];
+	const offsets = new Array<Node>();
```

Comment:
Does this change anything?

Reply:
No, it's a leftover from when I tried to specify array length, but decided to remove it as the improvement was minimal. I will change this to `[]`.